ibm_mq/tests/test_ibm_mq_int.py

```diff
@@ -314,3 +314,26 @@ def test_stats_metrics(aggregator, get_check, instance):
 
     assert_all_metrics(aggregator)
     aggregator.assert_metrics_using_metadata(get_metadata_metrics())
+
+
+def test_check_custom_dynamic_queue(aggregator, get_check, instance_dynamic_queue):
+    # Ensure that the custom dynamic tag is being used as reply-to queue instead of the default one
+    instance_dynamic_queue['mqcd_version'] = os.getenv('IBM_MQ_VERSION')
```

Comment:
How difficult would it be to have a test env that creates a dynamic queue and then runs the check. Also, what happens in this test specifically when the check tries to use a queue that does not exist?